components/account.tsx

```
@@ -0,0 +1,40 @@
+import { useAccount, useDisconnect, useEnsAvatar, useEnsName } from 'wagmi'
+import { useChainId } from 'wagmi'
+
+const Account = () => {
+  const chainId = useChainId()
+  const { address } = useAccount()
+  const { disconnect } = useDisconnect()
+  const { data: ensName } = useEnsName({ address })
+  const { data: ensAvatar } = useEnsAvatar({ name: ensName! })
```

ESLint

[error] 'ensAvatar' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)